### ISSUE TITLE
More Functions in Formulae

### ISSUE THREAD

**Issue by theFlyingCat04** [tgrosinger/advanced-tables-obsidian#290]
As a developer I use tables for recording Programm execution times. To get some statistics about two different versions of the program it would be nice to get minimum and maximum values of separate columns, as well as mean and median values. E. g. Id like to have a Table like 

| times   | old in seconds | new in seconds |
| ------- | -------------- | -------------- |
|         | 0.638          | 0.520          |
|         | 0.628          | 0.511          |
|         | 0.647          | 0.513          |
|         | 0.623          | 0.518          |
|         | 0.625          | 0.517          |
| Min     |          |                |
| Max     |                |                |
| Average |                |                |
| Median  |                |                |

and the last four rows beeing automatically calculated by formulas.

So to sum it up I would like to have statistic functions "min", "max", "median" in order to be able to do something like 
```
<!-- TBLFM: @7$2=min(@2$2..@6$2) -->
<!-- TBLFM: @8$2=max(@2$2..@6$2) -->
<!-- TBLFM: @9$2=mean(@2$2..@6$2) -->
<!-- TBLFM: @10$2=median(@2$2..@6$2) -->
```

**Comment by theFlyingCat04** [comment]
Any Progress, Comments, Reactions on this? Are You planning to implement features like this or will i have to calculate these things by hand in the future?
